.travis.yml

```diff
@@ -1,5 +1,10 @@
+cache:
+  directories:
+    - $HOME/.m2
```

Comment:
So the theory behind starting to do this is that we are going to only be doing mvn verify below and that should ensure that build artifacts from previous builds never get cached?